src/main/java/impl/weather/MonthInfo.java

```diff
@@ -0,0 +1,52 @@
+package impl.weather;
+
+import api.weather.DayTemperatureInfo;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class MonthInfo {
+    private final Map < Integer, DayTemperatureInfo > info = new LinkedHashMap < > ();
```

Comment:
code-style!!!! I won't close it but it is the last try. I told you: just use "Reformat code" action. It will help in 99% cases